src/backend/controllers/wallet-controller.ts

```diff
@@ -0,0 +1,23 @@
+import { NextRequest } from "next/server";
+import { WalletService } from "../services/wallet";
+import {
+  HttpBadRequestHandler,
+  HttpCreatedHandler,
+} from "../helpers/httpExceptionHandler";
+
+export const WalletController = () => {
+  const { findByUserId } = WalletService();
```

Comment:
const { findByUserId: findWalletByUserID } = Wallet service();
Will be more readable.